packages/cli/src/__tests__/commands/DebugInfo.test.ts

```diff
@@ -330,6 +331,98 @@ describe('debug', () => {
     `)
   })
 
+  it('should read the .env file if it exists', async () => {
+    ctx.fixture('dotenv-debug-cmd')
+
+    // To make sure the terminal is always detected
+    // as non interactive, locally and in CI
+    process.env.TERM = 'dumb'
+
+    const result = await DebugInfo.new().parse([])
+
+    expect(result).not.toContain('this_is_private')
+    expect(result).toContain('from_env_file')
+
+    expect(cleanSnapshot(result as string)).toMatchInlineSnapshot(`
+      -- Prisma schema --
+      Path: REDACTED_PATH
+
+      -- Local cache directory for engines files --
+      Path: REDACTED_PATH
+
+      -- Environment variables --
+      When not set, the line is dimmed and no value is displayed.
+      When set, the line is bold and the value is inside the \`\` backticks.
+
+      For general debugging
+      - CI: \`true\`
+      - DEBUG: \`prisma*\`
```

Comment:
Seems this is already set when the .env file is loaded, hence has a different value than set in there.